src/option.c

```diff
@@ -6736,6 +6736,8 @@ win_copy_options(win_T *wp_from, win_T *wp_to)
     void
 after_copy_winopt(win_T *wp)
 {
+    // make sure w_leftcol is correct
+    (void)did_set_wrap(NULL);
```

Comment:
This doesn't use the `wp` passed in when the calls below all use it.

Reply:
ah thanks, obviously right